### PR TITLE
PS-8618 Apply new branding to docs (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/_static/Percona_Logo_Color.png
	new file:   docs/_static/percona-favicon.ico
	new file:   docs/_static/percona-favicon.png
	new file:   docs/_static/percona-favicon.svg
	modified:   docs/_static/percona-logo.svg
	modified:   docs/css/percona.css
	modified:   mkdocs-base.yml

### CODE REVIEW

File: mkdocs-base.yml [hunk 1]
```diff
@@ -15,11 +15,11 @@ use_directory_urls: false
 # Theme setup
 theme:
   name: material
-  logo: _static/percona-logo.svg
+  logo: _static/percona-logo.png
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml]: change to _static/percona-logo.svg

Comment (patrickbirch) [reply mkdocs-base.yml]: done

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: mkdocs-base.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ theme:
   favicon: _static/percona_favicon.ico
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml]: Change to favicon: _static/percona-favicon.ico

Comment (patrickbirch) [reply mkdocs-base.yml]: Done.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**